examples/export-avatars-positions.js

```diff
@@ -0,0 +1,147 @@
+
+//Created by Teodoro Dannemann --->  https://github.com/teo523    
+//Credits to Zach Capalbo ---> https://github.com/zach-capalbo/hubs-client-bot
+
+//This example uses hubs-bot for extracting (x,y,z) positions of every avatar connected to the room.
+//You can combine this with sockets.io to stream data to clients (see the following link 
+// where I used (x,y) position to create a live map of the room) 
+// ----> https://github.com/teo523/hubs-client-bot/tree/metapoiesis
+
+const {HubsBot} = require('../index.js')
+const fs = require('fs')
+
+
+class ImportExportBot extends HubsBot {
+  constructor({onlyPinned} = {}) {
+    super({
+      headless: true,
+      autoLog: false,
+    })
+    this.onlyPinned = onlyPinned
+  }
+  
+  //First wait to access the room
+  async accessRoom(room) {
+    await this.enterRoom(room)
+ 
+    // Wait for network sync. TODO: Add an actual event handler
+    await this.page.waitFor(5000);
+    
+
+    return this;
+  }
+
+  //Get (x,y,z) coordinates of connected avatars
+  async exportFromRoom() {
+    
+    let objects = this.evaluate( async function prueba(onlyPinned) {
+      var a = "";
+      var b = [];
```

Comment:
It would be good to have clearer names for these. I think something like

`var a` -> `var avatarEntity`
`var b` -> `var avatarPositions`

would make this function a lot easier to understand for people just learning this code